File: design-patterns/go/observer/general/observer_one.go
// observer_one.go

package main

import "fmt"

type ObserverOne struct {
	Observer
}

func NewObserverOne(subject Subject) (observerOne *ObserverOne) {
	observerOne = &ObserverOne{}
	observerOne.subject = subject
	observerOne.subject.Attach(observerOne)
	return
}

func (observerOne *ObserverOne) SendUpdate() {
	fmt.Printf("Received in ObserverOne: %v\n", observerOne.subject.GetState())
}

File: design-patterns/php/observer/general/ObserverOne.php
<?php
// ObserverOne.php

namespace BigBoxCode\DesignPattern\Observer\General;

class ObserverOne extends Observer {
    public function __construct(Subject $subject) {
        $this->subject = $subject;
        $this->subject->attach($this);
    }

    public function sendUpdate(): void {
        echo "Received in ObserverOne: " . $this->subject?->getState() . "\n";
    }
}